presentation/src/main/java/me/androidbox/presentation/event/screen/EventScreen.kt

```diff
@@ -86,11 +94,43 @@ fun EventScreen(
                     )
 
                     Spacer(modifier = modifier.height(26.dp))
+                    AgendaDropDownMenu(
+                        modifier = Modifier
+                            .background(color = MaterialTheme.colorScheme.dropDownMenuBackgroundColor),
+                        shouldOpenDropdown = eventScreenState.shouldOpenDropdown,
+                        onCloseDropdown = {
+                            eventScreenEvent(
+                                EventScreenEvent.OnShowAlarmReminderDropdown(shouldOpen = false))
+                        },
+                        listOfMenuItemId = listOf(
+                            R.string.ten_minutes_before,
+                            R.string.thirty_minutes_before,
+                            R.string.one_hour_before,
+                            R.string.six_hours_before,
+                            R.string.one_day_before),
+                        onSelectedOption = { item ->
+                            eventScreenEvent(EventScreenEvent.OnAlarmReminderTextChanged(
+                                AlarmReminderItem.values()[item].text))
+
+                            eventScreenEvent(EventScreenEvent.OnShowAlarmReminderDropdown(shouldOpen = false))
+
+                            /** TODO Using mock data to set the Alarm Monitor, here use the eventScreenState start time date */
+                            val alarmItem = AlarmItem(
+                                ZonedDateTime.now().plusSeconds(30L),
+                                "THIS IS THE MESSAGE FROM THE ALARM REMINDER"
+                            )
```

Comment:
Just using mock data here, but in my next PR I will be using the start time/date that was sent for the event.